### ISSUE TITLE
Possibility to add a class to a menu container

### ISSUE THREAD

**Issue by daevid66** [isaacplmann/ngx-contextmenu#106]
I have several context menus in my app and want to style one of them differently (making items with icons stack horizontally). I cannot style this since there is no way to add a class to the elements containing the menu.

I think this would be useful for a lot of other scenarios as well.

Keep up the good work!

**Comment by daevid66** [comment]
OK, I'll see what I can do..!

**Comment by daevid66** [comment]
So as you'll see I'm a noob when it comes to forking/pull requests. 

First problem - I get the error message: 
ERROR in ./src/polyfills.ts
Module not found: Error: Can't resolve 'zone.js/dist/zone' in '...\repos\ngx-contextmenu\src'

My steps:
- forked the repo
- run npm install
- ng serve (I assume this is how you run the demo). 

I tried installing zone manually but still get the same error message.

**Comment by daevid66** [comment]
I'm in the root, and ng start doesn't work (The specified command start is invalid).
This is the version trace:
Angular CLI: 1.7.3
Node: 6.10.1
OS: win32 x64
Angular: 5.2.9
... common, compiler, compiler-cli, core, platform-browser
... platform-browser-dynamic, router

@angular/cdk: 5.2.4
@angular/cli: 1.7.3
@angular-devkit/build-optimizer: 0.3.2
@angular-devkit/core: 0.3.2
@angular-devkit/schematics: 0.3.2
@ngtools/json-schema: 1.2.0
@ngtools/webpack: 1.10.2
@schematics/angular: 0.3.2
@schematics/package-update: 0.3.2
typescript: 2.4.2
webpack: 3.11.0

**Comment by isaacplmann** [comment]
Try `npm start` not `ng start`.  I'm using node 8.x, but I'm not sure if that is causing your problem.

**Comment by isaacplmann** [comment]
I just got the same error after blowing away my node_modules.  Working on a fix.

**Comment by isaacplmann** [comment]
Try getting latest and running `npm install` and then `npm start` again.  `zone.js` wasn't specified as a dependency.  I'm not sure how the build ever worked.

**Comment by daevid66** [comment]
I don't get any changes when pulling master, what branch are you working on? I should get your latest changes even when I've forked, right?

**Comment by isaacplmann** [comment]
Nope.  Your master points daevid66/ngx-contextmenu.  Here's what you want to do:

`git remote add upstream https://github.com/isaacplmann/ngx-contextmenu.git`
`git fetch upstream` // this will pull down the latest from the upstream remote
`git checkout master` // if you're not on master already
`git merge upstream/master` // merges my repo with your fork's master

Then when you make changes, do it on a branch of your own master.  That way you will never have to conflict resolve between your master and the upstream master.  The conflict resolution will happen on your feature branch.

Let me know if that makes sense.

**Comment by daevid66** [comment]
Thanks for your help, now I got it running. Now let's see if I can add anything useful :)

Do I understand it correctly that I should work on a feature branch, not on master?

**Comment by isaacplmann** [comment]
The feature branch is just a suggestion to make your life easier.  It
doesn't matter from my perspective.

**Comment by isaacplmann** [comment]
And you've actually already added something useful by finding that bug in the build process.  Now the next person who comes along to submit a PR will have an easier time.

**Comment by daevid66** [comment]
Added a pull request, let me know what you think...

/D
